main.go

```diff
@@ -39,6 +40,7 @@ func init() {
 	flag.BoolVar(&breakingOnly, "breaking-only", false, "display breaking changes only")
 	flag.StringVar(&format, "format", formatYAML, "output format: yaml, text or html")
 	flag.BoolVar(&failOnDiff, "fail-on-diff", false, "fail with exit code 1 if a difference is found")
+	flag.BoolVar(&version, "version", false, "show version and quit")
```

Comment:
Doing it this way standardizes the ```version``` flag so it behaves like all other flags.

Reply:
works for me